packages/core/src/routes/applications/application.ts

```diff
@@ -213,6 +213,11 @@ export default function applicationRoutes<T extends ManagementApiRouter>(
       }
 
       ctx.body = application;
+
+      if (rest.type === ApplicationType.MachineToMachine) {
```

Comment:
Should we set the body content after the quota report?

Reply:
Like we discussed offline, the user do not care whether the report of subscription updates is successful or not, they only care if the resource is created/deleted as expected. Also we have caught the potential error of reporting subscription updates (if there is any) and we should debug the error on Cloud side.